src/locale/locales/fr/messages.po

```diff
@@ -2277,7 +2168,7 @@ msgstr "ex. Artiste, amie des chiens et lectrice passionnée."
 
 #: src/lib/moderation/useGlobalLabelStrings.ts:43
 msgid "E.g. artistic nudes."
-msgstr "Ex. nus artistiques."
+msgstr "Nus artistiques par exemple."
```

Comment:
i feel like i've always heard "nudité artistique" over "nus artistiques", i think that'd be a better fit